src/style.css

```
@@ -1 +1,75 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  --active-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding-right: 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 50px;
+}
+
+.nav__list {
+  display: flex;
+  padding-inline-start: 0;
+  margin: 0;
+}
+
+.list-item {
+  display: flex;
+}
+
+.link {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  padding: 23px 0;
+  margin-left: 20px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.is-active {
+  color: var(--active-color);
+  margin-left: 0;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  color: #00acdc;
+  content: attr(data-descr);
+  display: flex;
+  position: absolute;
+  height: 4px;
+  top: 56px;
+  min-width: 37px;
```

```suggestion
  width: 100%;
```
and add `position: relative;` to `.link`